modules/statics/src/coins.ts

```diff
@@ -54,8 +54,15 @@ export const coins = CoinMap.fromCoins([
     ETH2_FEATURES,
     KeyCurve.BLS
   ),
-  account('etc', 'Ethereum Classic', Networks.main.ethereum, 18, UnderlyingAsset.ETC, ETH_FEATURES),
-  account('tetc', 'Testnet Ethereum Classic', Networks.test.kovan, 18, UnderlyingAsset.ETC, ETH_FEATURES),
+  account('etc', 'Ethereum Classic', Networks.main.ethereumClassic, 18, UnderlyingAsset.ETC, ETH_FEATURES),
+  account(
```

Comment:
pre-commit script keeps formatting the indentation sigh!

Reply:
yes, that's the idea.

Reply:
I would have also tried to format other lines too haha! I was mainly worried if anyone will ask me to change the formatting back :-)